h2o-core/src/main/java/water/HeartBeat.java

```diff
@@ -48,6 +48,31 @@
   public long get_pojo_mem() { return _pojo_mem; }
   public long get_free_mem() { return _free_mem; }
   public long get_swap_mem() { return _swap_mem; }
+  
+  private long _total_mem;
```

Comment:
the HeartBeat structure should be kept minimal, most of this information is static and never changes. It is thus not a good idea to broadcast it with every single heartbeat.